tests/e2e/lib.sh

```diff
@@ -146,6 +146,7 @@ export_test_environment() {
     ci_export ROX_VULN_MGMT_2_GA "${ROX_VULN_MGMT_2_GA:-true}"
     ci_export ROX_VULN_MGMT_ADVANCED_FILTERS "${ROX_VULN_MGMT_ADVANCED_FILTERS:-true}"
     ci_export ROX_VULN_MGMT_LEGACY_SNOOZE "${ROX_VULN_MGMT_LEGACY_SNOOZE:-true}"
+    ci_export ROX_VULN_MGMT_NO_CVES_VIEW "${ROX_VULN_MGMT_NO_CVES_VIEW:-true}"
```

Comment:
Although the incremental addition of this feature will degrade the UX for an end user, it should not break functionality in CI.